HHS/OPRE-OPS
# Ops 1228/table row errors

## What changed

Adds table row errors if validation fails on budget lines.

## Issue

#1228 

## How to test

1. create agreement with incomplete/invalid budget lines
2. see the errors on the agreement review page

## Screenshots

_If relevant, e.g. for a front-end feature_
